src/Components/ProvisioningWizard/steps/ReservationProgress/index.js

```diff
@@ -52,21 +53,24 @@ const ReservationProgress = ({ setLaunchSuccess }) => {
     },
   });
 
+  const reservationParams = () => ({
+    source_id: chosenSource,
+    [instanceType(provider)]: chosenInstanceType,
+    amount: chosenNumOfInstances,
+    image_id: chosenImageID,
+    [region(provider)]: chosenRegion,
+    pubkey_id: chosenSshKeyId,
+    launch_template_id: chosenTemplate,
```

Comment:
This is just for AWS, I think the parameter would be safely ignored for other providers, but I wonder if we want to address this by having params function that would accept provider as parameter?